### PR TITLE
Plotbot purge

### PR DESCRIPTION
**Pull request by scjody**
While I'm in here... getting rid of the images Plotbot posts, as discussed a while ago with the team.

@bpostlethwaite please review.

### CODE REVIEW

File: totw/totw.go [hunk 1]
```diff
@@ -17,27 +17,12 @@ func init() {
...
-		"http://uproxx.files.wordpress.com/2012/09/iron.gif",
-	})
 	plotbot.RegisterStringList("tech adept", []string{
 		"you're a real tech adept",
 		"what an investigator",
 		"such deep search!",
 		"a real innovator you are",
 		"way to go, I'm impressed",
-		"hope it's better than my own code",
-		"noted, but are you sure it's good ?",
```
Comment (scjody) [inline totw/totw.go]: @alexander-daniel This one's for you!

**Comment by chriddyp** [comment]
ahhhh thank youuuuuu

**Comment by theengineear** [comment]
\o/

**Comment by scjody** [comment]
@perigee I forgot you've also worked on Plotbot.  If you feel comfortable reviewing this please do so.;

**Comment by bpostlethwaite** [comment]
:dancer:

**Pull request merged**